### PR TITLE
chore: update linter config

### PR DESCRIPTION
**Pull request by xcoulon**
disable 'golint', use 'revive' instead

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Review by MatousJobanek — APPROVED** [review]

File: controllers/toolchainstatus/toolchainstatus_controller.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ const (
 	hostOperatorTag        statusComponentTag = "hostOperator"
 	memberConnectionsTag   statusComponentTag = "members"
 	counterTag             statusComponentTag = "MasterUserRecord and UserAccount counter"
-	minutesAfterUnready    time.Duration      = 10
+	minutesAfterUnready    time.Duration      = 10 * time.Minute
```
Comment (alexeykazakov) [inline controllers/toolchainstatus/toolchainstatus_controller.go:67]: Let's rename it. It's not the number of minutes actually. It's a duration.
durationAfterUnready?

File: controllers/usersignup/usersignup_controller.go [hunk 2]
```diff
@@ -548,7 +548,7 @@ func (r *Reconciler) DeleteMasterUserRecord(mur *toolchainv1alpha1.MasterUserRec
 
 	err := r.updateStatus(logger, userSignup, inProgressStatusUpdater)
 	if err != nil {
-		return nil
+		return err
```
Comment (alexeykazakov) [inline controllers/usersignup/usersignup_controller.go:551]: I guess we are missing a test for this case? Otherwise we would catch it earlier.

**Review by alexeykazakov — APPROVED** [review]
Looks good. Just a minor comment.

File: controllers/usersignup/usersignup_controller.go (hunk 2 above)
Comment (xcoulon) [reply controllers/usersignup/usersignup_controller.go:551]: ah right, I intended to add a test for that, as I did in https://github.com/codeready-toolchain/member-operator/pull/328/files#diff-f57f1b01b263fd00b64ab1ec2d007285e03675feb420c7732360f639b38ce736R1262 when making a similar change, but I forgot about it 🤦‍♂️ ! Thanks for the reminder ;)

Comment (xcoulon) [reply controllers/usersignup/usersignup_controller.go:551]: done in 1af071a by adding a test inspired from `TestUserSignupDeactivatedButMURDeleteFails` just above

File: controllers/toolchainstatus/toolchainstatus_controller.go (hunk 1 above)
Comment (xcoulon) [reply controllers/toolchainstatus/toolchainstatus_controller.go]: done in fd6f00a

**Pull request merged**